index.css

```diff
@@ -0,0 +1,175 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300&display=swap');
+@import url(./Authentication/authentication.css);
+@import url(./Wishlist/wishlist.css);
+@import url(./productlisting/productlisting.css);
+@import url(./Cartmanagement/Cartmanagment.css);
+
+:root{
+    --black: #121212;
+    --white:#ffffff;
+    --whitesmoke:whitesmoke;
+    --primary:#4110F8;
+    --red:#e02522;
+    --lightBlue: #78909C;
+  }
+
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif;
+}
+
+body{
+    background-color: var(--whitesmoke);
+ 
+}
+
+.rr-nav-container{
+    background-color:var(--whitesmoke);
+    display: flex;
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+    border-bottom: solid 1px black;
+    box-shadow: 0px 1px 2px 2px var(--primary);
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: auto;
+    z-index: 3;
+}
+
+.header-logo{
+    width:5.625rem;
```

Comment:
Why such a specific value? Try keeping max 1 decimal point

Reply:
Will fix it. Thank you.